bindings/python/examples/how_tos/accounts_and_addresses/check_balance.py

```diff
@@ -17,4 +17,4 @@
 
 # Just calculate the balance with the known state
 balance = account.get_balance()
-print(f'Balance {json.dumps(balance, indent=4)}')
+print(f'Balance {json.dumps(balance.as_dict(), indent=4)}')
```

Comment:
`as_dict()` needs to be manually defined and can for example convert something, `__dict__` is automatically derived and will not do any conversions

Reply:
Should this be able to use `__dict__` then?

Reply:
Not in `json.dumps()`, because there are other classes inside of this one and then it returns an error `Object of type BaseCoinBalance is not JSON serializable` unless we would add a custom encoder
It's possible to print without the `json.dumps()`/`.as_dict()`, but then the output wouldn't be the same JSON format as we have also in the other languages